### DISCUSSION TITLE
Use `changelogs` in code and docs

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#23109] [Suggest an Idea]
### Tell us more.

Our docs and code uses `release notes` _and_ `change logs`.

Is there a difference in _meaning_ that I'm missing here? Is changelog more like a short list? And release notes like a short blog of what, why and how of the changes?

If there's no big difference in meaning, we could try to standardize to a single term? 😄 

Let me know what you think!

**Comment by viceice** [comment]
should be same, so unify them is a good idea 💡👍

**Reply by rarkins** [reply]
Changelogs seems more generic. Release notes more typically refers to _features_

**Reply by HonkingGoose** [reply]
@rarkins can you pick a option from the list?

1. Standardize on changelogs in code and docs 
2. Standardize on releasenotes in code and docs
3. Keep things as they are now
4. Something else

**Reply by rarkins** [reply]
(1) and we should have a note in our docs so someone searching for the term release notes finds an explanation that we use the term changelogs

**Reply by HonkingGoose** [reply]
I made two issues that will resolve this discussion. See:

- #23139 
- #23140
